liquibase-core/src/main/java/liquibase/datatype/core/IntType.java

```diff
@@ -72,6 +72,9 @@ public DatabaseDataType toDatabaseDataType(Database database) {
         if (database instanceof SybaseASADatabase) {
             return new DatabaseDataType("INTEGER");
         }
+        if (database instanceof SybaseDatabase) {
```

Comment:
Similar question to the one in BigIntType.java. Can this be moved to be an "or" in the "if" at line 65?

Reply:
It can but doesn't really matter. I moved it since I was taking a couple things out otherwise.